userspace/sysdig/plugin_utils.cpp

```diff
@@ -21,7 +21,18 @@ limitations under the License.
 
 #include "plugin_utils.h"
 
+#include <utility>
+
+struct PluginLoaded
+{
+	PluginLoaded(string& path, string& init_conf) : path(path), init_config(init_conf)
+	{}
+	string path;
+	string init_config;
+};
+
 vector<plugin_dir_info> g_plugin_dirs;
+map<string, shared_ptr<PluginLoaded>> g_loaded_plugins;
```

Comment:
I'm not convinced about the `shared_ptr<>` here, it seems to me that there can only be one instance of a `PluginLoaded` which is owned by the `std::map`, so wouldn't a `map<string, PluginLoaded>` be enough?